### ISSUE TITLE
Merged stories get "breaking news" added for no reason

### ISSUE THREAD

**Issue by soytakyon** [SoylentNews/rehash#287]
Merged stories get a breaking news label added. Numerous editors have forgotten to remove this after merging a story. One easy solution would be to apply all nexuses and topics from the 2+ submissions to the merged submission. So if one submission has Main Page and Science, another has Main Page, Science, and Software, and the last has Breaking News and Hardware, both nexuses and all three topics get added.

**Comment by paulej72** [comment]
OK it may be that it is no longer merging topics properly.   I'll need to look at the code and see.

**Comment by paulej72** [comment]
OK it looks like story merge code only works on story content and notes, not any of the meta data.  It creates a new story that defaults to the defaulttopic in the vars.  On prod this is tid=1 or The Main Page.  

It should not be picking Breaking News.

**Comment by paulej72** [comment]
OK with only a  nexus selected, the topic select defaults to the first item, which is Breaking News.

I may be able to do a quick and dirty fix for this in lieu of doing a topic merge

**Comment by paulej72** [comment]
Quick and dirty fix is to set defaulttopic to something other than The Main Page as it is not a real topic (not set to display and not story pickable).  I suggest we set it to News (tid=6).   This seems to be the most all encompassing of the topics.  This can be done easily as it only requires a var change.

Merging of the topics looks like it will not be possible without a major update as the current db schema only allows for a single value to be stored for the topic of a submission, only once it has been promoted to a story does the db support having multiple topics.

We could just use the topic from one story as well.  Please comment.

**Comment by TheMightyBuzzard** [comment]
@soytakyon Need input.
